client/components/DateSelector/useStyles.js

```diff
@@ -23,16 +23,17 @@ const useStyles = makeStyles(theme => ({
     borderRight: `1.5px solid ${theme.palette.text.secondaryLight}`,
     height: '1.2rem',
   },
+  // Note: "Last Week", "Last Month" and Last 3 Months colors are defined in option.
```

Comment:
I'm curious why these comments are necessary, here and in all the other components? It seems like the color specifications are all commented out.

Reply:
Color specifications are commented out to see how the dark theme expresses itself without any custom overrides. The comments are necessary to keep track of what part of the `FilterMenu` each custom override affects. For example, if we want to change the background color of the `Request Types` section, a comment gives us a reference to where to accomplish this quickly if we need to

Reply:
I see, so this is not intended to be checked in. It's just to show Melissa the "default" design, correct?

Reply:
Exactly. I think starting from "default", seeing what can be done directly from theme.js and then only if absolutely necessary, adding any necessary overrides to achieve the v2 design makes sense. 

In the meanwhile, I'm just making earmarks in certain places so we are able to add back any necessary overrides to the FilterMenu easily if needed. There are so many different areas presently. 

Once everything is finalized, we could definitely get rid of any/all comments but I think the `Notes` are still helpful to future frontend maintainers because it allows them to know what part of the front-end a particular background or color property affects.

Also, I am staging these PR incrementally and strategically so reviews do not get too overwhelming (as too much to at a time can easily lead to overlooked bugs). Doing little bits at a time.